src/System.Management.Automation/help/UpdatableHelpSystem.cs

```diff
@@ -419,6 +419,7 @@ private string ResolveUri(string baseUri, bool verbose)
                         using (HttpClient client = new HttpClient(handler))
                         {
                             client.Timeout = new TimeSpan(0, 0, 30); // Set 30 second timeout
+                            // codeql[cs/ssrf] - This is expected Poweshell behavior and the user assumes trust for the Uri and invocation is done on the user's machine, as PowerShell is not a web application/server. If there is concern for remoting, restricted remoting guidelines should be used.
```

Comment:
This is updatable help.  Let's just remove this suppression for now and leave it for another batch.